WalletWasabi.Documentation/CodeCoverage.md

```diff
@@ -1,12 +1,35 @@
-Install `altcover` NuGet to the `WalletWasabi.Tests` project and Coverage Glutters extension to vscode.
+# Code coverage (How to)
+
+Wasabi wallet is build using dotnet core. Given there is no a cross-platform Profiling API like the one available on Windows, we 
+use [AltCover](https://github.com/SteveGilham/altcover) package for instrumenting the assemblies and recording the execution
+coverage.
+
+So, first of all we need to install the package in the `WalletWasabi.Tests` project as follow:
+
+```sh
+dotnet add WalletWasabi.Tests/WalletWasabi.Tests.csproj package AltCover
+```
+
+Next, we must build the solution as usual. After that, it is necessary to instrument the assemblies, run the tests and collect coverage:
 
 ```sh
 dotnet build
-dotnet C:\Users\user\.nuget\packages\altcover\5.0.663\tools\netcoreapp2.0\altcover.dll --save --inplace "-i=bin\Debug\netcoreapp2.2"
+
+altcover=$(locate AltCover.dll | tail -1)
+dotnet $altcover --save --inplace "-i=./WalletWasabi.Tests/bin/Debug/netcoreapp2.2"
 dotnet test
-dotnet C:\Users\user\.nuget\packages\altcover\5.0.663\tools\netcoreapp2.0\altcover.dll runner --collect "-r=bin\Debug\netcoreapp2.2" "-l=lcov.info"
+dotnet $altcover runner --collect "-r=./WalletWasabi.Tests/bin/Debug/netcoreapp2.2" "-l=lcov.info"
+```
+
+All the previous steps can be simplified running just one line:
```

Comment:
Ok, I'm dead :D Can you remove the previous ones? Let's just add altcover to our test project so we can simplify even the "install nuget blah blah" lines.  
Also we could gitignore the generated files.

Reply:
Not sure how to do that manually, I use tortoise git, but nevermind, not important.